ssl/ssl_lib.c

```diff
@@ -2011,6 +2012,70 @@ int ssl_write_internal(SSL *s, const void *buf, size_t num, size_t *written)
         return s->method->ssl_write(s, buf, num, written);
     }
 }
+ossl_ssize_t SSL_sendfile(SSL *s, int fd, off_t offset, size_t size, int flags)
```

Comment:
Blank line before the start of this function

Reply:
Done